client.go

```diff
@@ -260,9 +260,6 @@ func runTCPLatencyTest(test *ethrTest) {
 	}
 	defer conn.Close()
 	buffSize := test.testParam.BufferSize
-	// TODO Override buffer size to 1 for now. Evaluate if we need to allow
-	// client to specify the buffer size in future.
-	buffSize = 1
```

Comment:
Are we happy that the tool works with this removed and doesn't cause any regressions? Have you tested it?

Reply:
@boyan-soubachov we noticed the packet buffering when we did tcpdumps while running the latency tests. I tested it out by recompiling w/ my changes and then ran a tcpdump to make sure we were seeing packets > 1b. I tested it with changes for 16b, 32b, 16kb, etc and it looked good

Reply:
The default is 16kb IIRC. In order to test anything > 16kb, we actually tested it by passing in the `-l` to the server. Our server start would look like ```ethr -s -6 -l 24kb````. To test the client we ran the following ```ethr -c ethr_server1 -t l -l 24kb -i 1```

Reply:
Can we make default to 1 for this test? That way we won't regress existing behavior. So for bandwidth default may be 16KB, but for Latency test, it would be 1B default.

Reply:
fair enough. let me think about a good way to do this so that latency tests defaults to 1 if a user doesnt pass in the actual buffer length on the command line. i'm traveling for work this week but i should have a PR sometime later this week with that change.

Reply:
Test result outcomes. I did this by running a tcpdump to verify the packet size
1. ethr -c agent1 -t l
```
18:11:37.337758 IP6 perf1 > agent1: Flags [P.], seq 165606:165607, ack 165772, win 224, options [nop,nop,TS val 2245845694 ecr 3486412573], length 1
```

2. ethr -c agent1 -t l -b 6KB
```
18:14:50.338157 IP6 perf1 > agent1 Flags [P.], seq 604356001:604362001, ack 603756001, win 1432, options [nop,nop,TS val 3486605574 ecr 2246038691], length 6000

```

3. ethr -c agent1 -t b 
18:16:35.337423 IP6 perf1 > agent1: Flags [.], seq 2944662129:2944726389, ack 1, win 225, options [nop,nop,TS val 3486710573 ecr 2246143689], length 64260
```

```